### PR TITLE
Improve WebView fragment

### PR DESCRIPTION
**Pull request by maniac103**
Supercedes #1252

### CODE REVIEW

**Comment by maniac103** [comment]
@mueller-ma Please give this a try. This includes your NPE fix + a fix to properly destroy the WebView when the fragment is killed off.

File: mobile/src/main/java/org/openhab/habdroid/ui/activity/WebViewFragment.java [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+            mWebview.destroy();
+            mWebview = null;
+        }
+    }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+        if (mWebview != null) {
+            mWebview.onResume();
```
Comment (mueller-ma) [inline mobile/src/main/java/org/openhab/habdroid/ui/activity/WebViewFragment.java]: Should call resumeTimers() as well.

File: mobile/src/main/java/org/openhab/habdroid/ui/activity/WebViewFragment.java [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+        }
+    }
+
+    @Override
+    public void onPause() {
+        super.onPause();
+        if (mWebview != null) {
+            mWebview.onPause();
```
Comment (mueller-ma) [inline mobile/src/main/java/org/openhab/habdroid/ui/activity/WebViewFragment.java]: Should call pauseTimers() as well.

File: mobile/src/main/java/org/openhab/habdroid/ui/activity/WebViewFragment.java [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+            }
+
+            @Override
+            public void onReceivedError(WebView view, int errorCode, String description,
+                                        String failingUrl) {
+                Log.e(TAG, "onReceivedError() (deprecated) on URL: " + failingUrl);
+                updateViewVisibility(true, false);
+            }
+        });
+        mWebview.setWebChromeClient(new WebChromeClient());
```
Comment (mueller-ma) [inline mobile/src/main/java/org/openhab/habdroid/ui/activity/WebViewFragment.java]: #1254 is missing here.

Comment (maniac103) [reply mobile/src/main/java/org/openhab/habdroid/ui/activity/WebViewFragment.java]: Rebase fail on my part, fixed.

**Pull request merged**